dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/AccessTokenController.java

```diff
@@ -74,20 +72,18 @@
     @PostMapping(value = "/create")
     @ResponseStatus(HttpStatus.CREATED)
     @ApiException(CREATE_ACCESS_TOKEN_ERROR)
-    @AccessLogAnnotation(ignoreRequestArgs = "loginUser")
-    public Result createToken(@ApiIgnore @RequestAttribute(value = Constants.SESSION_USER) User loginUser,
-                              @RequestParam(value = "userId") int userId,
+    @AccessLogAnnotation()
+    public Result createToken(@RequestParam(value = "userId") int userId,
                               @RequestParam(value = "expireTime") String expireTime,
                               @RequestParam(value = "token") String token) {
 
-        Map<String, Object> result = accessTokenService.createToken(loginUser, userId, expireTime, token);
+        Map<String, Object> result = accessTokenService.createToken(AuthUtils.getLoginUser(), userId, expireTime, token);
```

Comment:
What is the significance of this change?

Reply:
1. only get login user info on need rather than on every controller's handler.  2. reduce the parameter of the handler then make code concise. 3. remove the whole user info from the log if we need logging request.

Reply:
> and we do not need to pass user info to every service method. we can remove all service method user info parameters, and get user info on need.

These should be two different things. If the service really needs it, the Controller can display the declaration (not all Controllers need this parameter), and then pass it.

Reply:
there are too many login user parameters passed， this change remove all login user parameters make code concise。

Reply:
and remove `ignoreRequestArgs = "loginUser"` in `@AccessLogAnnotation(ignoreRequestArgs = "loginUser") `  of every controller